app/javascript/packs/home/home.js

```diff
@@ -0,0 +1,16 @@
+const FormElement = document.getElementById('form-text');
+const BtnElement = document.querySelectorAll(".btn");
+const CardElement = document.getElementsByClassName("card")
+const DeleteElement = document.getElementsByClassName(".form-group p")
+console.log(DeleteElement);
+
+
+
+document.addEventListener("DOMContentLoaded", function(){
```

Comment:
`turbolinks` の読み込み後にしないと正常に動作しません。

```suggestion
document.addEventListener("turbolinks:load", () => {
```